core/src/components/slides/slides.tsx

```diff
@@ -138,62 +141,78 @@ export class Slides {
     const finalOptions = this.normalizeOptions();
     // init swiper core
     this.swiper = new Swiper(this.container, finalOptions);
+    this.readyResolve(true);
+  }
+
+  /**
+   * Calls true if the swiper core is initialized
+   */
+  @Method()
+  ready(): Promise<boolean> {
```

Comment:
can you make this a private method without @Method()? we can expose it later

```
private waitUntilReady() {
```

also, can you move it to the bottom just before `normalizeOptions`?

Reply:
Of course - will commit it soon